mongodb-queue.js

```diff
@@ -266,3 +273,12 @@ Queue.prototype.done = function(callback) {
         callback(null, count)
     })
 }
+
+Queue.prototype._optionsWithNewDocument = function(query) {
+    if (this.returnDocument) {
+        query.returnDocument = 'after'
```

Comment:
This should probably be called `options` instead of `query` to be consistent with caller

Reply:
Yes good catch!